duckduckgo/iOS
# remove set as default statistics

Task/Issue URL: https://app.asana.com/0/1205278999335242/1205332386813734/f
Tech Design URL:
CC:

**Description**:

Removes set as default statistics.

**Steps to test this PR**:
1. Run the app, ensure no daily active user pixel or set as default pixel is fired. 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
